libbpf-tools/cpudist.c

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2020 Wenbo Zhang
+//
+// Based on cpudist(8) from BCC by Brendan Gregg & Dina Goldshtein.
+// 8-May-2020   Wenbo Zhang   Created this.
+#include <argp.h>
+#include <limits.h>
+#include <search.h>
+#include <signal.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <sys/resource.h>
+#include <unistd.h>
+#include <time.h>
+#include <bpf/libbpf.h>
+#include <bpf/bpf.h>
+#include "cpudist.h"
+#include "cpudist.skel.h"
+#include "trace_helpers.h"
```

Comment:
I bet some of these includes are not necessary?

Reply:
Yes, sorry for this. I'll check more next time.